docs/client.md

```diff
@@ -1,13 +1,16 @@
 ---
 title: Client API | ldapjs
-markdown2extras: tables
 ---
 
 # ldapjs Client API
 
+<div class="intro">
```

Comment:
Do we really need to introduce structural HTML into the Markdown?

Reply:
I was just trying to keep the style the same as it is now. We can change the style however we want. We should probably also remove the Joyent logo at some point. But this PR was mostly to get the website up to date with the docs changes. We can change the style in another PR.

Reply:
I'm not completely opposed. Just wondering if we think it is worth the noise in the docs.

Reply:
I think it is worth it for now but I'm ok with changing it in another PR.